salesforce_api/mysql-record-to-sfdc-new-product/readme.md

```diff
@@ -0,0 +1,49 @@
+Example to create a new product in Salesforce using MySQL and Salesforce integration.
+
+## Use case
+MySQL is an open-source relational database management system that can be easily used for data storing and retrieving applications. 
+
+Organizations maintain details about products in various data stores such as ERP systems, databases, etc. These data stores could be updated by different business units. It is important to keep Salesforce up to date about the organization's current product lineup so that sales staff can effectively promote and sell valid products.
+
+The following sample demonstrates a scenario of creating products in Salesforce with product details fetched from a MySQL database.
+
+## Prerequisites
+* Salesforce account
+* MySQL Client
+
+### Setting up a Salesforce account
+1. Visit [Salesforce](https://www.salesforce.com/) and create a Salesforce Account.
+2. Create a connected app and obtain the following credentials:
+    *   Base URL (Endpoint)
+    *   Access Token
+    *   Client ID
+    *   Client Secret
+    *   Refresh Token
+    *   Refresh Token URL
+3. When you are setting up the connected app, select the following scopes under Selected OAuth Scopes:
+    *   Access and manage your data (api)
+    *   Perform requests on your behalf at any time (refresh_token, offline_access)
+    *   Provide access to your data via the Web (web)
+4. Provide the client ID and client secret to obtain the refresh token and access token. For more information on obtaining OAuth2 credentials, go to [Salesforce documentation](https://help.salesforce.com/articleView?id=remoteaccess_authenticate_overview.htm).
+5. Once you obtained all configurations, Replace "" in the `Config.toml` file with your data.
```

Comment:
```suggestion
5.Suggested change
5. Once you obtained all configurations, Replace "" in the `Conf.toml` file with your data.
5. Once you obtained all configurations, add those to the `config.toml` file.

```

Reply:
Updated in [bcf09ac](https://github.com/ballerina-guides/integration-samples/pull/31/commits/bcf09ac42e87543485854586312facacacd050cb)